dashboard/test/ui/features/star_labs/can_see_finish.feature

```diff
@@ -35,35 +35,30 @@ Feature: Make sure we can see the finish button for all LEVEL TYPE levels on sma
   Scenario: can see finish button on "Minecraft Adventurer"
     And I check that the minecraft free play level for "Minecraft Adventurer" shows the finish button for small screens
 
-  # The following levels currently do not show finish button
-  # in the viewport when play is pressed for small screens.
-
-  # TODO: Fix this - https://codedotorg.atlassian.net/browse/LP-1318
-    # "Minecraft Heroes Journey"
-    # "Minecraft Designer"
-    # "App Lab"
-
-  @no_ie @no_safari @no_firefox @no_chrome
+  # A row of arrow control buttons result in the Finish button not being visible in Dance Lab on our phone simulator,
```

Comment:
This is a real issue on iPhone where you can't see the finish button on Dance Party. I'd rather get all of our iPhone tests on a more commonly used phone, then fix this bug (and re-enable this test).